flytekit/core/context_manager.py

```diff
@@ -617,7 +617,26 @@ def current_context() -> Optional[FlyteContext]:
         """
         return FlyteContextManager.current_context()
 
-    def get_deck(self) -> str:
+    def get_deck(self) -> typing.Union[str, "IPython.core.display.HTML"]:
```

Comment:
this is not going to work in mypy, right? In other words, this is purely for documentation purposes.

Reply:
yes

Reply:
should i delete it?

Reply:
At the very least we should leave a comment explaining that this is not mypy-compliant. 

How about we type-ignore this for now?
```suggestion
def _get_deck(
    new_user_params: ExecutionParameters,
) -> typing.Union[str, "IPython.core.display.HTML"]:  #type:ignore

```